easybuild/easyconfigs/r/R/R-4.1.2-foss-2021b.eb

```diff
@@ -3215,6 +3215,24 @@ exts_list = [
     ('catlearn', '0.8', {
         'checksums': ['0d13da431a5511b025090b7ffaa3765b94b33c7de5a7cc1cec8c50437755646e'],
     }),
+    ('MetaUtility', '2.1.2', {
+        'checksums': ['e38c21588c239aa8926e64d916aa0f3b04108c2992f0e801095e4c7920b9ad5d'],
+    }),
+    ('EValue', version, {
+        'checksums': ['d9f3feb3da2efaf4e5d794e05475be62666192a7c855e373c3d369324078690c'],
+    }),
+    ('ggdag', '0.2.4', {
+        'checksums': ['372ec4ef0ded9637304a19b50bce908e84a034e54dbd2a52a061bb747b2b08a0'],
+    }),
+    ('dagitty', '0.3-1', {
+        'checksums': ['7d44b5d259ec3fef776a7e3fcb21d1c379f930d6ae9ae5fbfff54494ad78e8a8'],
+    }),
+    ('metadat', '1.0-0', {
```

Comment:
@deniskristak `metadat` also needs to move up, above `MetaUtility`, to fix:

```
ERROR: dependency metadat is not available for package MetaUtility
```

Same for `dagitty`, which is required by `ggdag`.

This order works for me when not using `--parallel-extensions-install`:

```python
    ('metadat', '1.0-0', {
        'checksums': ['ed335a4eb2145fffde20e36077e360f9d36a615e9d93453d2bbd6f5967db4b9c'],
    }),
    ('MetaUtility', '2.1.2', {
        'checksums': ['e38c21588c239aa8926e64d916aa0f3b04108c2992f0e801095e4c7920b9ad5d'],
    }),
    ('EValue', version, {
        'checksums': ['d9f3feb3da2efaf4e5d794e05475be62666192a7c855e373c3d369324078690c'],
    }),
    ('dagitty', '0.3-1', {
        'checksums': ['7d44b5d259ec3fef776a7e3fcb21d1c379f930d6ae9ae5fbfff54494ad78e8a8'],
    }),
    ('ggdag', '0.2.4', {
        'checksums': ['372ec4ef0ded9637304a19b50bce908e84a034e54dbd2a52a061bb747b2b08a0'],
    }),
    ('simex', '1.8', {
        'checksums': ['80c7841196b9377a9367eb6960ad80ca0bf8de511b8b18a0031bfbe7bde289a0'],
    }),
```